agent/lib_guzzle6.c

```diff
@@ -352,8 +354,13 @@ NR_PHP_WRAPPER_START(nr_guzzle6_client_construct) {
 
   if (NRINI(vulnerability_management_package_detection_enabled)) {
     char* version = nr_php_get_object_constant(this_var, "VERSION");
+    if (NULL == version) {
+      version = nr_php_get_object_constant(this_var, "MAJOR_VERSION");
+    }
     // Add php package to transaction
-    nr_txn_add_php_package(NRPRG(txn), "guzzlehttp/guzzle", version);
+    nr_txn_add_php_package(NRPRG(txn), PHP_PACKAGE_NAME, version);
```

Comment:
doesn't only this function call need to be in the `if (NRINI(vulnerability_management_package_detection_enabled))` block?
We want version supportability metric whether vm is on or off.

Reply:
Fixed: [647d424](https://github.com/newrelic/newrelic-php-agent/pull/868/commits/647d424ffb824368d14f803f6812df43a75be043)